pkg/types/agent_config.go

```diff
@@ -26,6 +26,8 @@ var AgentEnvMapping = []EnvMapping{
 	{"ENV_CLUSTER_DNS_DOMAIN", "cluster.local", &AgentConfig.ClusterDnsDomain},
 	{"ENV_LOCAL_NODE_IP", "", &AgentConfig.LocalNodeIP},
 	{"ENV_LOCAL_NODE_NAME", "", &AgentConfig.LocalNodeName},
+	{"DNS_SERVICE_SELECT_LABEL_KEY", "kubernetes.io/name", &AgentConfig.DnsServiceSelectLabelKey},
```

Comment:
Unified format, use ENV as the beginning

Reply:
OK, environment variable prefixes have been unified